### PR TITLE
Training ResNet_50v1.5 on ImageNet

### PR DESCRIPTION
**Pull request by shauray8**
* Using SGD to train ResNet
* Implement cross-entropy Loss
* Adding Transforms
Should reach the desired accuracy within 50 epochs in theory ( Implementation details - https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Classification/ConvNets/resnet50v1.5 )
* SGD doesn't have weight decay???

### CODE REVIEW

**Comment by geohot** [comment]
Bounty locked, good luck!

**Comment by wozeparrot** [comment]
If you set `GPU=1` it should train on gpu

**Comment by shauray8** [comment]
I'm using Ubuntu right now on an Intel i5

**Comment by wozeparrot** [comment]
Do you have a dedicated gpu or are you just running off of the i5 igpu?

**Comment by shauray8** [comment]
I do have a dedicated Nvidia GPU

**Comment by wozeparrot** [comment]
You can try with `CUDA=1` or `TRITON=1` but I'm pretty sure triton is broke rn: #855

**Comment by shauray8** [comment]
Training on a relatively smaller dataset with a batch size of 8 (due to memory constraints), should be reproducible by `CUDA=1 python3 ./examples/mlperf/model_train`.

**Comment by geohot** [comment]
No update in a week
